ResearchKit/Consent/ORKConsentSection.m

```diff
@@ -169,6 +169,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
         ORK_DECODE_OBJ_CLASS(aDecoder, summary, NSString);
         ORK_DECODE_OBJ_CLASS(aDecoder, content, NSString);
         ORK_DECODE_OBJ_CLASS(aDecoder, htmlContent, NSString);
+        ORK_DECODE_OBJ_CLASS(aDecoder, contentRequest, NSURLRequest);
```

Comment:
Shouldn't have to *write* tests. Just make sure the tests pass. They'll fail if you add a property and do nothing.